lib/pbench/agent/tool_data_sink.py

```diff
@@ -221,7 +226,17 @@ def launch(self):
                 )
                 return 0
 
-            args = ["podman", "pull", "quay.io/prometheus/prometheus"]
+            try:
+                prom_reg = PbenchAgentConfig(
```

Comment:
Here you don't need the agent configuration directly, you should be able to use `prom_reg` environment variable.